Tests/TSCUtilityTests/DownloaderTests.swift

```diff
@@ -18,6 +18,7 @@ import FoundationNetworking
 #endif
 
 class DownloaderTests: XCTestCase {
+
```

Comment:
**nitpick:** I don't think the rest of the test cases use this kind of formatting.

```suggestion
```